### PR TITLE
Rewrite match `=` as `->` inside of `match`

### PR DESCRIPTION
**Pull request by V-FEXrt**
Updates `wake-format` rule to always emit `->` as the match case separator which shows the correct data flow direction.

### CODE REVIEW

File: tools/wake-format/emitter.cpp [hunk 1]
```diff
@@ -1591,15 +1591,13 @@ wcl::doc Emitter::walk_flag_global(ctx_t ctx, CSTElement node) {
 
 wcl::doc Emitter::walk_guard(ctx_t ctx, CSTElement node) {
   MEMO(ctx, node);
-  MEMO_RET(walk_placeholder(ctx, node));
```
Comment (ag-eitilt) [inline tools/wake-format/emitter.cpp:1594]: I assume `walk_placeholder` is used elsewhere and so doesn't become dead code?

File: tests/tests.wake
Comment (ag-eitilt) [inline tests/tests.wake:1]: This file is the only one which seems to be related to testing -- don't the unit/golden tests have to be updated to reflect this as well?

**Review by ag-eitilt — COMMENTED** [review]
So far as I'm able to read the formatter code, this looks good to me -- the `*.wake` changes certainly seem to be doing the right things.

File: tools/wake-format/emitter.cpp (hunk 1 above)
Comment (V-FEXrt) [reply tools/wake-format/emitter.cpp:1594]: yeah, its the main fallback function

File: tests/tests.wake
Comment (V-FEXrt) [reply tests/tests.wake:1]: they do, I missed them

**Review by JakeSiFive — APPROVED** [review]
let 'er rip

**Pull request merged**